libs/api/ebpf_api.cpp

```diff
@@ -1154,6 +1161,43 @@ ebpf_link_detach(_In_ struct bpf_link* link)
     EBPF_RETURN_RESULT(_detach_link_by_handle(link->handle));
 }
 
+ebpf_result_t
+ebpf_program_detach(
+    fd_t program_fd,
+    _In_ const ebpf_attach_type_t* attach_type,
+    _In_reads_bytes_(attach_parameter_size) void* attach_parameter,
+    _In_ size_t attach_parameter_size)
```

Comment:
```suggestion
    size_t attach_parameter_size) noexcept(false)
```
Don't use `_In_` with value parameters, only pointers.
And add noexcept(false) since this function can throw an exception (because it calls resize()).
However it would be better to implement this function to not throw an exception.

Reply:
THanks for catching the lack of try/catch block.